src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,87 @@
+package duke.ui;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+public class Ui {
+	private String name;
+	private final Scanner in;
+	private PrintStream out;
+	private static final String COMMENT_LINE_REGEX = "#.";
+
+	private static final String LOGO = " ____        _        \n"
```

Comment:
Try changing your logo to something else! Maybe try [this](https://www.twitchquotes.com/copypastas/3553). I think it fits your code really well.